lib/KCard/index.vue

```diff
@@ -167,6 +172,19 @@
         type: String,
         default: 'centerInside',
       },
+      /**
+       * Controls the alignment of the thumbnail area in horizontal layouts.
+       * Only applies to horizontal layouts with 'small' or 'large' thumbnail display.
+       * Ignored in other layouts.
+       * @type {String}
+       * @values 'left', 'right'
+       * @default 'left'
+       */
+      thumbnailAlign: {
+        type: String,
+        default: thumbnailAlign.LEFT,
+        validator: value => [thumbnailAlign.LEFT, thumbnailAlign.RIGHT].includes(value),
```

Comment:
Nothing blocking, just could be nice to use the `cardValidator` you implemented some time ago?